client/src/components/Reservation.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+// import ReactDOM from 'react-dom';
```

Comment:
the only comments that are merged in should be proper comments that describe the code. If you've commented out a line for debugging purposes etc, try to remove it before you push.